Widen main column

pjsier opened the pull request:
Fixes #315, if this isn't wide enough I can make it wider, but this seemed worthwhile given it's quick and the issues we had on #314 with column spacing

cecilia-donnelly commented:
This is great!  Thanks, @pjsier.  Can you make the `max-width` attribute even bigger; maybe 1200px?  On large screens there's no reason to restrict.  Actually, do we need `max-width` at all, since we have `width` set?

pjsier commented:
If it works, I just took the opposite approach and dropped `width` and increased the `max-width` so that it'll really fill up the screen. Added some padding on the sides too so that it doesn't look to cramped

cecilia-donnelly commented:
Works for me!
